backend/src/main/java/heartbeat/service/board/jira/JiraService.java

```diff
@@ -449,8 +456,10 @@ else if (customFieldValue.equals("Flagged") && !fieldValue.isJsonNull()
 		for (int index = 0; index < customFieldMapList.size(); index++) {
 			allCardsResponseDTO.getIssues().get(index).getFields().setCustomFields(customFieldMapList.get(index));
 		}
-		for (int index = 0; index < sprintList.size(); index++) {
-			allCardsResponseDTO.getIssues().get(index).getFields().setSprint(sprintList.get(index));
+
+		for (int index = 0; index < allCardsResponseDTO.getIssues().size(); index++) {
```

Comment:
store `allCardsResponseDTO.getIssues()` as a variable to reduce the calculate when running